recipes/coincurve/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set name = "coincurve" %}
+{% set version = "18.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c86626afe417a09d8e80e56780efcae3ae516203b23b5ade84813916e1c94fc1
```

Comment:
Please use the GitHub release instead of the PYPI souce dist so that you don't have to manually add the license files here. That way, if they change in the future release, they will automatically be updated.

Reply:
Should I systematically prefer github releases (I misinterpreted the doc thinking Pypi was preferred). How about a particular commit? I see you have done that in one of your recipes (I am trying to find best practice )?

Reply:
You can use the PYPI source dists if it includes all of the necessary artifacts, but in this case it doesn't. The developers of this package didn't think to include the license files in the PYPI source dist.

Reply:
Updated to using github commit. Resolved